subsystem/elevator.py

```diff
@@ -0,0 +1,72 @@
+import rev
+
+from toolkit import Subsystem
+from toolkit.motors.rev_motors import SparkMax, SparkMaxConfig
+
+import config
+
+# TODO: CHANGE WHEN ROBOT IS BUILT
+ELEVATOR_CONFIG = SparkMaxConfig(
+    0.055, 0.0, 0.01, 0.000, (-.5, .75), idle_mode=rev.CANSparkMax.IdleMode.kBrake
+)
+
+class Elevator(Subsystem):
+
+    motor_extend: SparkMax = SparkMax(
+        config.elevator_can_id, config=ELEVATOR_CONFIG, inverted=False
+    )
+
+    def __init__(self) -> None:
+        super().__init__()
+        # Absolute encoder
+        self.zeroed: bool = False
+
+    def init(self) -> None:
+        self.motor_extend.init()
+        self.encoder: rev.SparkAbsoluteEncoder = rev.SparkAbsoluteEncoder(self.motor_extend.motor) # TODO: Make sure params work
```

Comment:
This will probably be connected to the roborio, so I would actually switch the class to a WPILIB analog Encoder. Check out the feat/drivetrain config file for more on how that works.

Reply:
Actually - scratch that; there is no space on the Rio after the swerve encoders lol. Look at the #11 pull request, as that has an absolute encoder with the right methods in it.